tictactoe-starter-code/script.js

```diff
@@ -0,0 +1,21 @@
+var turn = "a"
+
+
+$('#reset').click(function(){
```

Comment:
If you wanted to make a function that would reset the board on click, you'd want to remove the classes from all the td elements, not from the reset button itself.